src/components/dropdown/dropdown.ts

```diff
@@ -368,8 +367,8 @@ export default class IgcDropdownComponent extends EventEmitterMixin<
   public navigateTo(value: string): IgcDropdownItemComponent | null;
   /** Navigates to the item at the specified index. If it exists, returns the found item, otherwise - null. */
   public navigateTo(index: number): IgcDropdownItemComponent | null;
+  /* blazorSuppress */
```

Comment:
Because it's no longer a decorator that resolves on each overload rather leading trivia that's separate, at least with the current impl that doesn't distinguish the main method you'll need to add `blazorSuppress` to the two overloads as well